modified_cdc_protocol/operation_types/rehydratepatientsample/protocol.rb

```diff
@@ -0,0 +1,171 @@
+# typed: false
+# frozen_string_literal: true
+
+needs 'Standard Libs/PlanParams'
+needs 'Standard Libs/Debug'
+needs 'Standard Libs/Pipettors'
+needs 'Standard Libs/Units'
+needs 'Standard Libs/AssociationManagement'
+
+needs 'Collection Management/CollectionDisplay'
+needs 'Collection Management/CollectionTransfer'
+
+needs 'Diagnostic RT-qPCR/DiagnosticRTqPCRHelper'
+
+needs 'Tube Rack/TubeRack'
+
+needs 'Modified CDC Protocol/RehydratePatientSampleDebug'
+
+TEMPLATE = 'Template'
+MEDIA = 'Media'
+
+EPP_LABEL = 'Eppendorf Tube'
+
+class Protocol
+  # Standard Libs
+  include PlanParams
+  include Debug
+  include Pipettors
+  include Units
+  include AssociationManagement
+  include PartProvenance
+
+  #Collection Management
+  include CollectionDisplay
+  include CollectionTransfer
+
+  #Diagnostic RT-qPCR
+  include DiagnosticRTqPCRHelper
+
+  #TubeRack
+  include TubeRackHelper
+
+  #Modified CDC Protocol
+  include RehydratePatientSampleDebug
+
+  attr_reader :job_params, :operation_params, :media_volume
+  def default_operation_params
+    {
+      media_volume: 300,
```

Comment:
If you do this as `media_volume_microliters: 300` you can use [this](https://github.com/klavinslab/standard-libraries/blob/235ec592f1900dbc89cc9fc88398d803df1a47fc/libraries/Units.rb#L65) to convert to a standardized representation.